### ISSUE TITLE
wrong linting with line-break on @click

### ISSUE THREAD

**Issue by cs09g** [vuejs/eslint-plugin-vue#1051]
**Tell us about your environment**	
* **ESLint version:** 5.9.0
* **eslint-plugin-vue version:** 6.1.2
* **Node version:**	10.13.0

**Please show your full configuration:**

```json
module.exports = {
    extends: [
        "google",
        "eslint:recommended",
        "plugin:vue/recommended",
    ],
    parserOptions: {
        ecmaVersion: 2018,
        parser: "babel-eslint",
        sourceType: "module",
    },
    env: {
        browser: true,
        amd: true,
        node: true,
    },
    rules: {
        "linebreak-style": 0,
        "quotes": ["error", "double"],
        "object-curly-spacing": ["error", "never"],
        "no-restricted-globals": ["error", "event"],
        "camelcase": ["error", {"properties": "never", "ignoreDestructuring": true}],
        "max-len": ["error", {"code": 120, "ignoreUrls": true, "ignoreStrings": true}],
        "no-trailing-spaces": ["error"],
    }
};
```

**What did you do?**	

Typed following and save
```vue
<li
  @click="$emit('click')"
>
```

**What did you expect to happen?**
should be staying as it is or should be in a single line.

**What actually happened?**

Turn to wrong syntax
```vue
<li
  @click=" 'click')"
>
```
in case you will wonder vscode user setting
```json
{
    "workbench.colorTheme": "Monokai",
    "editor.tabSize": 2,
    "javascript.format.insertSpaceAfterFunctionKeywordForAnonymousFunctions": false,
    "javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false,
    "terminal.integrated.shell.windows": "C:\\Windows\\System32\\cmd.exe",
    "[javascript]": {},
    "files.associations": {
        "*.js": "javascript",
        "*.vue": "vue",
        "*.scss": "scss"
    },
    "editor.fontWeight": "200",
    "typescript.validate.enable": false,
    "javascript.validate.enable": false,
    "css.validate": false,
    "scss.validate": false,
    "javascript.updateImportsOnFileMove.enabled": "always",
    "[html]": {
        "editor.defaultFormatter": "vscode.html-language-features"
    },
    "typescript.updateImportsOnFileMove.enabled": "always",
    "editor.renderWhitespace": "all",
    "debug.node.autoAttach": "on",
    "diffEditor.ignoreTrimWhitespace": false,
    "eslint.format.enable": true,
    "editor.codeActionsOnSave": {
        "source.fixAll": true
    },
    "eslint.run": "onSave",
    "files.trimTrailingWhitespace": true,
    "files.eol": "\n"
}
```

**Comment by ota-meshi** [comment]
Thank you for this issue.

I can't reproduce this issue.
Do you get the same issue if you run eslint with CLI?
Also, could you share your minimal repository?

**Comment by ota-meshi** [comment]
If you still have problems, please share the repository to reproduce the problem.
